getting_started/configuration_file.md

```diff
@@ -64,6 +67,26 @@ project.
 }
 ```
 
+### Using configuration file as an import map
+
+```jsonc
+{
+  "imports": {
+    "std/": "https://deno.land/std@0.174.0/"
+  },
+  "scopes": {
+    // Use "old" version of standard library for source code inside "legacy/"
+    // directory
+    "legacy/": {
+      "std/": "https://deno.land/std@0.120.0/"
+    }
+  },
+  "tasks": {
```

Comment:
I feel like tasks should go first similar to how scripts is above dependencies in package.json?